src/jira/client/jira-client.ts

```diff
@@ -348,6 +367,22 @@ const withinIssueKeyLimit = (resources: { issueKeys: string[] }[]): boolean => {
 	return Math.max(...issueKeyCounts) <= ISSUE_KEY_API_LIMIT;
 };
 
+/**
+ * Returns if the max length of the issue key field is within the limit
+ * Assumption is that the transformed resource only has one association which is for
+ * "issueIdOrKeys" association.
+ */
+const withinIssueKeyAssociationsLimit = (resources) => {
+	let issueKeyCounts = [];
+
+	if (resources?.associations?.length > 0) {
+		issueKeyCounts = resources.map((resource) => resource.associations[0].values.length);
+		return Math.max(...issueKeyCounts) <= ISSUE_KEY_API_LIMIT;
+	}
+
+	return issueKeyCounts;
```

Comment:
Shouldn't this return `true` and not an array?

Reply:
Oops yeah nice catch. Typings would've helped hehe